### PR TITLE
fixing maps plugin cli vars

### PR DESCRIPTION
**Pull request by djett41**

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -99,6 +99,8 @@
     <!-- ios -->
     <platform name="ios">
         <preference name="API_KEY_FOR_IOS" />
+        <preference name="LOCATION_WHEN_IN_USE_DESCRIPTION" default="Show your location on the map"/>
+        <preference name="LOCATION_ALWAYS_USAGE_DESCRIPTION" default="Trace your location on the map" />
```
Comment (Anu2g) [inline plugin.xml:103]: Do these strings ever show up anywhere? We definitely do not want these strings showing up anywhere

**Comment by abhisekshukla** [comment]
I think these where the string that where getting updated on info.plist
earlier when we had issue with apple approval.

**Comment by Anu2g** [comment]
Right. I believe devin's intent here is that, when we do trax build rea or trax build hurdlr, it will pull the correct strings from the config_rea or config_hurdlr xml files. But just wanted to verify with @djett41

File: plugin.xml (hunk 1 above)
Comment (djett41) [reply plugin.xml:103]: these are the variables

**Comment by djett41** [comment]
Yeah this fixes that issue.  The maps plugin right now has the vars prefixed with a $ but they arent UPPER_CASE_WITH_UNDERSCORE which is required for cordova CLI vars to work.  so this PR allows us to add the vars to the config.xml, and the plugin will inject them.  This is the same change I pushed up to the main repo, waiting for them to merge it in, then we could prob get rid of this repo or fork it at their commit

**Pull request merged**